torch/csrc/stable/ScalarType.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+/*
```

Comment:
Could you extend the comment to describe the rationale for this file (iiuc that the below mirrors  `AT_FORALL_SCALAR_TYPES_WITH_COMPLEX_AND_QINTS` and `toString` for ScalarType but the set of ScalarTypes supported by torch/csrc/stable is a subset of that supported by libtorch)

Reply:
The stable `toString` support is now in torch core, so this comment and `toString` implementation is obsolete here, I'll remove these.